src/search/search-bar.tsx

```diff
@@ -63,8 +64,19 @@ export const SearchBar = (): JSX.Element => {
 	);
 	const [inputTyped, setInputTyped] = useState<string>('');
 	const history = useHistory();
-	const { updateQuery, module, query, searchDisabled, setSearchDisabled, tooltip } =
-		useSearchStore();
+	const {
+		updateQuery,
+		module: currentSearchModuleRoute,
+		query,
+		searchDisabled,
+		setSearchDisabled,
+		tooltip
+	} = useSearchStore();
+	const modules = useAppStore((s) => s.views.search);
+	const module = useMemo(
```

Comment:
It seems like this module is calculated just for the label at line 298. Am I missing something? does it make sense to do it just for that?

Reply:
We could extract and memoize the label directly. I did a first wrong implementation were I was using module also for the other checks (replacing the currentSearchModuleRoute), but I reverted this change since it was breaking things.

We need the translated label of the module, and for now the only place where we can find it without a more complex refactor, is inside the search views of the appStore